response_operations_ui/controllers/cir_controller.py

```diff
@@ -22,6 +22,16 @@ def get_cir_service_status():
     return _get_response_content(app.config["CIR_API_URL"] + "/status")
 
 
+def get_cir_metadata(survey_id, formtype):
+    # The CIR API requires both classifier_type and language to be present as part of the call in order to obtain the
+    # Relevant survey with form type. After speaking with Alina, as RASRM only deals with English surveys
```

Comment:
We probably don't need to include this. It's probably a well enough understood to include the default language en.